databox/examples/example.org.ts

```diff
@@ -1,7 +1,7 @@
 // NOTE: you must start your own Ulixee Miner to run this example.
 
-import Databox from '@ulixee/databox-for-hero';
-import { string } from '@ulixee/schema';
+import Databox, { schema } from '@ulixee/databox-for-hero';
```

Comment:
I find the lowercased `schema` to be a bit confusing. Why isn't treated as a static `Schema` class?

Reply:
I should clarify. I added a convenience "re-export" of schema from Databox, DataboxForHero and DataboxForPuppeteer

Reply:
Changed to Schema